.github/workflows/reviewdog.yml

```diff
@@ -0,0 +1,44 @@
+name: reviewdog-suggester
+
+on:
+  pull_request: {}
+
+jobs:
+  black:
```

Comment:
Sorry, could you rename this?

Reply:
So sorry for bothering you... [c48d396](https://github.com/optuna/optuna/pull/3357/commits/c48d396645557f610e602f645004a4d077b305f0) fixed.